bitrot.go

```diff
@@ -137,6 +137,8 @@ func (cmd *Generate) Execute(args []string) (err error) {
 
 	manifestPath := filepath.Join(manifestDir, manifestFile.Filename)
 	if _, err := os.Stat(manifestPath); os.IsNotExist(err) {
+		err = os.MkdirAll(manifestDir, 0755)
```

Comment:
We were already doing this on line 121 so I'm a little suspicious that this does anything...